app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% @contestants.each do |contestant| %>
+  <%= tag.h2 contestant.name %>
```

Comment:
You can wrap your erb snippets instead of using `tag.h2`

For example: 
`<h2><%= contestant.name %></h2>`